app/services/onlinetesting/Phase1TestService.scala

```diff
@@ -126,6 +126,13 @@ trait Phase1TestService extends OnlineTestService with ResetPhase1Test {
     registerAndInviteForTestGroup(application, getScheduleNamesForApplication(application))
   }
 
+  override def processNextExpiredTest(expiryTest: TestExpirationEvent)(implicit hc: HeaderCarrier, rh: RequestHeader): Future[Unit] = {
```

Comment:
Probably you can have a generic processNextExpiredTest in OnlineTestService, I guess the logic will be the same in all phases, the only change would be the repo and service that you call. If you like it, I think you can skip for now, and deal with it in another sprint.